src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/GenerateRevocationOrderController.kt

```diff
@@ -19,9 +18,7 @@ import java.util.Base64
 class GenerateRevocationOrderController(
   @Autowired private val pdfDocumentGenerator: PdfDocumentGenerator
 ) {
-  companion object {
-    val log: Logger = LoggerFactory.getLogger(this::class.java)
-  }
+  private val log = LoggerFactory.getLogger(this::class.java)
```

Comment:
not a biggie but I think the reason why it was in a companion object is so that the log field is static.